views/pictures.py

```diff
@@ -0,0 +1,52 @@
+import os
+import werkzeug
+from flask_restful import Resource, reqparse
+from models import Picture
+import http.client as client
+import uuid
+
+IMAGE_FOLDER = 'images'
+ALLOWED_EXTENSION = ['.jpg', '.jpeg', '.png', '.gif']
+
+
```

Comment:
Ho l'impressione che tutta questa serie di endpoint sia un work in progress. Suggerisco allora di fare una pr separata e nel frattempo fixare i problemi al codice degli Item